ocs_ci/framework/conf/default_config.yaml

```diff
@@ -110,6 +110,8 @@ REPORTING:
     project_id: "OpenShiftContainerStorage"
   # Upstream: 'US' or Downstream: 'DS', used only for reporting (Test Run Name)
   us_ds: 'DS'
+  ocs_must_gather_collection: true
```

Comment:
Those values should be documented in the documentation config file:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md - Currently what I see , those values are not configurable by user as if I will set both to false, it will be overwritten in autouse fixture collect_mg_rules .  So in this case we should also document it that those values are not suppose to be configured by user.